server/models/notes.js

```diff
@@ -0,0 +1,32 @@
+const mongoDB = require('../config/mongoDB')
+
+const note = mongoDB.Schema({
+    slug:{
+        type: String,
+        required: [true, "smalid is required"]
+    },
+    title: {
+        type: String,
+        required: [true, "title is required"]
+    },
+    notes : {
+        type: String,
+        required: [true, "note is required"]
+    },
+    created_at: {
+        type: Date,
+        required: [true, "date is required"]
+    },
+    updated_at: {
+        type: Date,
+        required: [true, "date is required"]
+    },
+    deleted:{
+        type: Boolean,
+        required: [true, "boolean required"]
+    }
+
+})
+
+
+module.exports = mongoDB.model('NotesSchema', note)
```

Comment:
For the model name, just give `Note` and name the schema as `NoteSchema`.

Reply:
ok

Reply:
done